mantle/kola/tests/coretest/core.go

```diff
@@ -38,11 +40,32 @@ var offServices = []string{
 }
 
 func init() {
+
 	register.RegisterTest(&register.Test{
-		Name:        "basic",
-		Description: "Verify basic functionalities like SSH, systemd services, useradd, etc.",
-		Run:         LocalTests,
-		ClusterSize: 1,
+		Name:             "basic.bios",
+		Description:      "Verify basic functionalities like SSH, systemd services, useradd, etc.",
+		ExcludeFirmwares: []string{uefi, uefiSecure},
+		Run:              LocalTests,
+		ClusterSize:      1,
+		NativeFuncs: map[string]register.NativeFuncWrap{
+			"PortSSH":        register.CreateNativeFuncWrap(TestPortSsh),
+			"DbusPerms":      register.CreateNativeFuncWrap(TestDbusPerms),
+			"ServicesActive": register.CreateNativeFuncWrap(TestServicesActive),
+			"ReadOnly":       register.CreateNativeFuncWrap(TestReadOnlyFs),
+			"Useradd":        register.CreateNativeFuncWrap(TestUseradd),
+			"MachineID":      register.CreateNativeFuncWrap(TestMachineID),
+			"RHCOSGrowpart":  register.CreateNativeFuncWrap(TestRHCOSGrowfs, []string{"fcos"}...),
+			"FCOSGrowpart":   register.CreateNativeFuncWrap(TestFCOSGrowfs, []string{"rhcos"}...),
+		},
+	})
+
+	register.RegisterTest(&register.Test{
```

Comment:
There's nothing constraining this test to `uefi` only and not e.g. `bios`.

Similarly for the tests below.

Reply:
Also, shouldn't they be constrained to QEMU only?

Reply:
Yikes; yes I have since fixed this now.